test/app.js

```diff
@@ -16,7 +16,7 @@ describe('generator-react-up TestComponent', function () {
 
   it('creates files', function () {
     assert.file([
-      '__test__/index.js',
+      '__test__/TestComponentSpec.js',
```

Comment:
Yeah, I have to figure out re-wiring the generator if we want to write it, in ES6.